### PR TITLE
org.tabos.banking

### PR DESCRIPTION
**Pull request by janbrummer**
FinTS online banking application for designed for Linux smartphones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Flathub
```
Comment (bilelmoussaoui) [inline README.md:1]: You probably want to rebase on top of new-pr branch to remove the unnecessary files & commits history.

File: org.tabos.banking.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            ]
+        },
+        {
+            "name" : "requests",
+            "buildsystem" : "simple",
+            "build-options" : {
+                "build-args" : [
+                    "--share=network"
+                ]
+            },
```
Comment (bilelmoussaoui) [inline org.tabos.banking.json:52]: Use https://github.com/flatpak/flatpak-builder-tools/tree/master/pip instead. Network access is not allowed at build time

File: README.md (hunk 1 above)
Comment (janbrummer) [reply README.md]: Yeah, that looked strange. Rebased it.

File: org.tabos.banking.json (hunk 2 above)
Comment (janbrummer) [reply org.tabos.banking.json]: Thanks for the hint, nice tool. Updated my PR.

**Comment by bilelmoussaoui** [comment]
bot, build org.tabos.banking

**Comment by barthalion** [comment]
Metainfo does not validate:

```
builddir/files/share/appdata/org.tabos.banking.appdata.xml: FAILED:
• tag-missing           : <release> required
```

**Comment by janbrummer** [comment]
Yeah, already fixing it.

**Comment by janbrummer** [comment]
bot, build org.tabos.banking

**Pull request closed without merging**